### ISSUE TITLE
Find workaround for inconsistent handling of Latex in Google Colab output cells

### ISSUE THREAD

**Issue by gutow** [gutow/Algebra_with_Sympy#34]
In Colab 
```
from IPython.display import HTML
display(HTML('$<latex expression>$<some following string>'))
```
does not render the `<latex expression>` as typeset math. Jupyter does.
It does appear that 
```
from IPython.display import Markdown
display(Markdown('$<latex expression>$<some following string>'))
```
renders properly in Colab. Thus, it might work to wrap the expression in `Markdown()` if that does not break Jupyter.

**Comment by gutow** [comment]
An alternative that seems to work in Colab is to drop the trailing string. In this case the label. This would require including a test to see if the code is running in Colab. This can be done by checking for a valid import of the form `from colab import output`.

**Comment by gutow** [comment]
Work around included in v1.0.2
